test/user-test.js

```diff
@@ -114,10 +79,7 @@ describe('User', function() {
       dailyStepGoal: 7000,
       friends: [1, 2, 3]
     });
-    const users = [user1, user2, user3, user4];
-    const userRepo = new UserRepo(users);
-
-    console.log(user2.getFriendsNames(userRepo));
-    expect(user2.getFriendsNames(userRepo)).to.deep.equal(['Alex Roth', 'The Rock', 'Rainbow Dash']);
+  
+    expect(user2.getFriendsNames(userRepo)).to.deep.equal(['Luisa Hane', 'Herminia Witting', 'Mae Connelly']);
```

Comment:
Unless it's supposed to randomize

Reply:
Yes, After looking at this again, the method isn't written right. 
The method invokes `getDataFromID()` but it's trying to pass in a name 
friends is an array of ids and has no name. so when I try test that function without those names, it comes back with error: `cannot read name of undefined `

Reply:
I just had to change one of the friends in the Sample data file since the userRepo in the before each only has access to the first 4 users
I was able to get rid of the alex, rock and rainbow users in that test.

<img width="872" alt="Screen Shot 2020-07-19 at 7 52 04 PM" src="https://user-images.githubusercontent.com/61163743/87892095-6367ad00-c9f9-11ea-956d-054058396d62.png">

Reply:
<img width="457" alt="Screen Shot 2020-07-19 at 7 53 49 PM" src="https://user-images.githubusercontent.com/61163743/87892169-9c078680-c9f9-11ea-9d28-753ab02a8a0b.png">

changed friend to 3 on line 12 (was 8)

Reply:
Awesome!